frontends/web/src/routes/settings/settings.tsx

```diff
@@ -195,6 +207,21 @@ class Settings extends Component<Props, State> {
                           <FiatSelection />
                         </div>
                         <div className="column column-2-3">
+                          <div>
+                            <h3 className="subTitle">{t('settings.info.title')}</h3>
+                            <div className="box slim divide m-bottom-large">
+                              { version !== undefined && update !== undefined ? (
+                                <SettingsButton
+                                  optionalText={version}
+                                  secondaryText={update ? t('settings.info.out-of-date') : t('settings.info.up-to-date')}
+                                  icon = {update ? RedDot({}) : Checked({})}
```

Comment:
this can be written with jsx `icon={update ? <RedDot /> : <Checked />}`

Reply:
Could you try to also add `disabled={!update}`?

Reply:
Ok for the `icon` part. 
Using `disabled` as-is was feasible, since it was bringing some CSS changes on the opacity too. I tried something different, making `disabled` a ternary variable with values `true`, `false` or `noclick`. This also allowed to remove the `hideButtonIcon` prop.